kernel/main.c

```diff
@@ -144,12 +144,6 @@ void stop_kernel(void)
 	struct proc *p, *next;
 	int n;
 
-	/* stop and disable all interrupts! */
-	CLI();
-	for(n = 0; n < NR_IRQS; n++) {
-		disable_irq(n);
-	}
-
```

Comment:
Just wondering what was the problem with Linux to have enabled interrupts. Does this is part of the Linux boot specification?

Reply:
I just checked and the specifications says to disable interrupts with `cli` before booting Linux.

I move the disable back to where it was and Linux appears to boot OK so I am not sure what the problem was before. I am running a full test under live-bootstrap (which takes over an hour) to see if this code can remain where it was. However, please see the comment below for another reason I moved the code.